src/components/Card.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from "react";
+
+const Card = ({
+  title,
+  image,
+  organization,
+  description,
+  price,
+}: Readonly<{
+  title: string;
+  image: string;
+  organization: string;
+  description: string;
+  price: string;
+}>): ReactElement => {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg m-1">
+      <img
+        src="https://gttp.imgix.net/225058/x/0/bohol-balicasag-balicasag-island-natanielluperte-2020-wm-1.jpg"
+        alt=""
+        className="w-full"
+      />
+      <div className="px-6 py-6 center">
+        <p className="tour-title text-lg">
+          <strong> Baler, Aurora Tour Package</strong>
+        </p>
+        <p className="tour-organization text-md text-gray-600">
+          {organization}
+        </p>
+        <p className="text-justify truncate ">{description}</p>
+        <div className="py-5 flex w-full ">
```

Comment:
```suggestion
        <div className="py-5 flex w-full">
```
@FutureHat  Para san yung mga extra space? :thinking:

Reply:
@efrenmercadojr may binura ata akong code dyan boss and hindi nasama space